app/pages/organizations/organization-container.jsx

```diff
@@ -90,7 +92,21 @@ class OrganizationContainer extends React.Component {
     }
 
     organization.get('projects', query)
-      .then(organizationProjects => this.setState({ fetchingProjects: false, organizationProjects }))
+      .then((organizationProjects) => {
+        this.setState({ fetchingProjects: false, organizationProjects });
+        const avatarRequests = organizationProjects
+          .filter(project => project.links.avatar.id)
+          .map(project => apiClient.type('avatars').get(project.links.avatar.id)
+            .catch(error => console.error('error loading project avatar', error))); // eslint-disable-line no-console)
```

Comment:
I don't think we should care if the response is a 404 and right now this will catch that also. We'll want to catch other errors. Maybe a conditional on the error.status?

Reply:
Makes sense and I think I updated accordingly. I thought I could test on [mborgtest](https://org-projects-request-refactor.pfe-preview.zooniverse.org/organizations/markb-panoptes/mborgtest) with the project without an avatar, but I'm not sure where I'd see the 404. Might not be relevant there though.

Reply:
nevermind. the filter is preventing the project without the avatar from being included in the `.map`.